components/TestimonyCard/TestimonyItem.tsx

```diff
@@ -21,17 +21,20 @@ import { UserInfoHeader } from "./UserInfoHeader"
 type FooterButtonProps = Omit<ButtonProps, "children"> & {
   className?: string
   children?: ReactNode
+  onClick?: any // Cannot find name 'MouseEventHandler'
```

Comment:
I think `MouseEventHandler` needs to be imported from `"react"` - we also use it here: https://github.com/codeforboston/maple/blob/5e2c3aa3ea2c64a2410aa5c0c40bd9acaa65a37d/components/TestimonyCard/Tabs.tsx#L2

Reply:
🚀